src/resources/postcss/base/forms/_text.pcss

```diff
@@ -5,29 +5,34 @@
 	 *
 	 * Example:
 	 * <div class="tribe-common-form-control-text">
-	 *   <label for="textInput">Text Input</label>
-	 *   <input id="textInput" name="textInput" type="text" placeholder="Text Input" />
+	 *   <label class="tribe-common-form-control-text__label" for="textInput">Text Input</label>
+	 *   <input
+	 *     class="tribe-common-form-control-text__input"
+	 *     id="textInput"
+	 *     name="textInput"
+	 *     type="text"
+	 *     placeholder="Text Input"
+	 *   />
 	 * </div>
 	 *
 	 * ----------------------------------------------------------------------------- */
 
-	.tribe-common-form-control-text {
+	/* .tribe-common-form-control-text {} */
 
-		label {
-			@mixin visually-hide;
-		}
+	.tribe-common-form-control-text__label {
+		@mixin visually-hide;
+	}
 
-		input[type="text"] {
-			@mixin desktop-body-2-bold;
+	.tribe-common-form-control-text__input {
```

Comment:
Are we handling focus styles for form inputs?

Reply:
looks like not, will check with matt

Reply:
still in flux he says, will update once it comes out ;)